boot.js

```diff
@@ -96,1994 +96,304 @@ function boot(preloaded, params) {
 
 }
 
-}],["asap","asap",{},function (require, exports, module){
+}],["asap","browser-asap",{"./raw":2},function (require, exports, module){
 
-// asap asap
```

Comment:
New version of Q upgrades ASAP and gets rid of its dependency on Collections in favor of depending directly on weak-map and pop-iterate.